mixins/rtl-mixin.js

```diff
@@ -13,9 +14,25 @@ export const RtlMixin = dedupeMixin(superclass => class extends superclass {
 
 	constructor() {
 		super();
+		this._localeSettings = getDocumentLocaleSettings();
+		this._handleLanguageChange = this._handleLanguageChange.bind(this);
+		this._handleLanguageChange();
+	}
+
+	connectedCallback() {
+		super.connectedCallback();
+		this._localeSettings.addChangeListener(this._handleLanguageChange);
```

Comment:
Would these listeners only get invoked if the language changes after the page loads? i.e. in a normal page where the language doesn't change, would `_handleLanguageChange` only get called once, from the `constructor`?

I'm not sure how concerned we should be about the number of handlers wired up. I can imagine pages easily wiring up hundreds due to the number of components, perhaps even more than a thousand. If the change handler is only needed for pages such as the demo page where the language can change, we could do this wire-up conditionally based on some global flag that the demo page sets. Ideally we can just wire it up and not worry about it though.

Reply:
Correct, these only get called if the language changes after the page loads. That currently only happens in the demo page (and maybe a few of our tests), although it's not impossible to foresee a scenario where we may allow the user to switch languages without reloading the whole app. Pretty edge case-y though.

I'm not especially worried about the add/remove of the listeners... it's just [appending to an array and then splicing it](https://github.com/BrightspaceUI/intl/blob/main/lib/common.js#L129. Agreed though that we could do some kind of global flag if it becomes an issue.

What I'm hoping though is that our dependence on `RtlMixin` will actually start to drop off now that we're in the clear to start using CSS logical properties. I'm guessing 95% of our uses are related to that.